frontend/src/components/navigation/Header.tsx

```diff
@@ -37,3 +37,43 @@ function Header() {
 }
 
 export default Header;
+*/
+
+import React, { Component } from 'react';
+import { MenuItems } from "./MenuItems"
+//import { Button } from "../Button"
+import './Header.css'
+
+//class Navbar extends Component {
+class Header extends Component {
+    state = { clicked: false }
+
+    handleClick = () => {
+        this.setState({ clicked: !this.state.clicked })
+    }
+
+    render() {
+        return(
+            <nav className="NavbarItems">
+                <h1 className="navbar-logo">OFTW @ Penn!{/*<i className="fab fa-react"></i>*/}</h1>
+                <div className="menu-icon" onClick={this.handleClick}>
+                    <i className={this.state.clicked ? 'fas fa-times' : 'fas fa-bars'}></i>
+                </div>
+                <ul className={this.state.clicked ? 'nav-menu active' : 'nav-menu'}>
+                    {MenuItems.map((item, index) => {
+                        return (
+                            <li key={index}>
+                                <a className={item.cName} href={item.url}>
+                                {item.title}
+                                </a>
+                            </li>
+                        )
+                    })}
+                </ul>
+                {/*<Button>Sign Up</Button>*/}
+            </nav>
+        )
```

Comment:
This largely looks good! Just be sure to convert this into using styled components instead.